app/models/form_template_related_document.rb

```diff
@@ -1,3 +1,6 @@
+# TODO this is too long, rename just to RelatedDocument as is sk.gov.schemas.servicebusserviceprovider.ness.eformprovider._1.RelatedDocument
 class FormTemplateRelatedDocument < ApplicationRecord
   belongs_to :form_template
+
+  # TODO rename document_type to type
```

Comment:
`type` sa pobije s AR STI

Reply:
napadlo ma, no `inheritance_column` vies zmenit, ale je to zatial TODO